packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -338,8 +338,11 @@ class EditableText extends StatefulWidget {
   /// If this is 1 (the default), the text will not wrap, but will scroll
   /// horizontally instead.
   ///
-  /// If this is null, there is no limit to the number of lines. If it is not
-  /// null, the value must be greater than zero.
+  /// If this is null, there is no limit to the number of lines, and the text
+  /// contianer will start with enough vertical space for one line and
+  /// automatically grow to accomodate additional lines as they are entered.
+  ///
+  /// If it is not null, the value must be greater than zero.
```

Comment:
If you're going to describe the sizing behavior, you might as well describe what the height will be when not null.